client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,25 @@
-import React from 'react';
+import React from "react";
 
 const MovieCard = props => {
-  return;
+  const {title, director, metascore, stars} = props.movie;
```

Comment:
Good job refactoring this into a separate component. You will see this pattern often when it comes to react. Creating new components to prevent code duplication